.gitignore

```diff
@@ -29,3 +29,4 @@ scripts/salt/*.minion
 /etc-salt-master
 
 .*.flag
+.bin/
```

Comment:
what do you guys have living in local `./bin` ?

Reply:
That was at my suggestion for the older terraform and vault binaries needed for builder as you wouldn't be able to use the new versions from homebrew packages, see the osx.md doc. It's only a suggestion, but I added to the gitignore just in case. Happy to remove if it's a problem and just leave the suggestion and for people to self manage.

Reply:
it's fine, just curious.